### PR TITLE
[Documentation] survey_accounts Test Plan updated

### LINKED ISSUE #6585: [survey_accounts] Test Plan can be updated and added to for clarity
The [test plan](https://github.com/aces/Loris/blob/23.0-release/modules/survey_accounts/test/TestPlan.md) for survey_accounts can be edited for accuracy and permissions checks. Some extra steps can be added, such as:

1. Once a new survey is created, check that the Survey List is updated. 
2. Check that a survey cannot be created for a Project that the user is not involved in. 

The formatting of the document can also be edited.

### PR DESCRIPTION
**Pull request by AlexandraLivadas**
The Test Plan for the survey_accounts module was updated to include additional steps (Project permissions and adding a survey). The formatting was also changed.

* Resolves #6585

### CODE REVIEW

**Comment by christinerogers** [comment]
@AlexandraLivadas is this ready for review?  Please request from @laemtl and me if so...

File: modules/survey_accounts/test/TestPlan.md [hunk 1]
```diff
@@ -14,26 +14,28 @@
...
-    The email can be pre-populated for each instrument using participant_emails table.
-    c. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).
-    d. Use the `Create Survey` button (no email address should be specified).
-    e. Use the `Email survey` button by specifying email address.
-    f. Try mismatched PSCID and DCCID and should get an error message.
-    g. Try creating duplicate instrument for a candidate for a visit, should get an error message.
-    h. Try creating instrument for a candidate in a non-existing visit, should get an error message.
-    [Manual Testing]
+    The email can be pre-populated for each instrument using participant_emails table.  
+    c. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).
```
Comment (laemtl) [inline modules/survey_accounts/test/TestPlan.md:23]: 3. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).

File: modules/survey_accounts/test/TestPlan.md [hunk 2]
```diff
@@ -14,26 +14,28 @@
...
-    c. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).
-    d. Use the `Create Survey` button (no email address should be specified).
-    e. Use the `Email survey` button by specifying email address.
-    f. Try mismatched PSCID and DCCID and should get an error message.
-    g. Try creating duplicate instrument for a candidate for a visit, should get an error message.
-    h. Try creating instrument for a candidate in a non-existing visit, should get an error message.
-    [Manual Testing]
+    The email can be pre-populated for each instrument using participant_emails table.  
+    c. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).  
+    d. Use the `Create Survey` button (no email address should be specified).
```
Comment (laemtl) [inline modules/survey_accounts/test/TestPlan.md:24]: 4. Use the `Create Survey` button (no email address should be specified).

File: modules/survey_accounts/test/TestPlan.md [hunk 3]
```diff
@@ -14,26 +14,28 @@
...
-    d. Use the `Create Survey` button (no email address should be specified).
-    e. Use the `Email survey` button by specifying email address.
-    f. Try mismatched PSCID and DCCID and should get an error message.
-    g. Try creating duplicate instrument for a candidate for a visit, should get an error message.
-    h. Try creating instrument for a candidate in a non-existing visit, should get an error message.
-    [Manual Testing]
+    The email can be pre-populated for each instrument using participant_emails table.  
+    c. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).  
+    d. Use the `Create Survey` button (no email address should be specified).  
+    e. Use the `Email survey` button by specifying email address.
```
Comment (laemtl) [inline modules/survey_accounts/test/TestPlan.md:25]: 5. Use the `Email survey` button by specifying email address.

File: modules/survey_accounts/test/TestPlan.md [hunk 4]
```diff
@@ -14,26 +14,28 @@
...
-    e. Use the `Email survey` button by specifying email address.
-    f. Try mismatched PSCID and DCCID and should get an error message.
-    g. Try creating duplicate instrument for a candidate for a visit, should get an error message.
-    h. Try creating instrument for a candidate in a non-existing visit, should get an error message.
-    [Manual Testing]
+    The email can be pre-populated for each instrument using participant_emails table.  
+    c. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).  
+    d. Use the `Create Survey` button (no email address should be specified).  
+    e. Use the `Email survey` button by specifying email address.  
+    f. Check that the survey list shows the newly created survey with all the correct information.
```
Comment (laemtl) [inline modules/survey_accounts/test/TestPlan.md:26]: 6. Check that the survey list shows the newly created survey with all the correct information.

File: modules/survey_accounts/test/TestPlan.md [hunk 5]
```diff
@@ -14,26 +14,28 @@
...
-    f. Try mismatched PSCID and DCCID and should get an error message.
-    g. Try creating duplicate instrument for a candidate for a visit, should get an error message.
-    h. Try creating instrument for a candidate in a non-existing visit, should get an error message.
-    [Manual Testing]
+    The email can be pre-populated for each instrument using participant_emails table.  
+    c. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).  
+    d. Use the `Create Survey` button (no email address should be specified).  
+    e. Use the `Email survey` button by specifying email address.  
+    f. Check that the survey list shows the newly created survey with all the correct information.    
+    g. Try creating a survey for a candidate in a Project that the user is not affiliated with. This should not be allowed.
```
Comment (laemtl) [inline modules/survey_accounts/test/TestPlan.md:27]: 7. Try creating a survey for a candidate in a Project that the user is not affiliated with. This should not be allowed.

File: modules/survey_accounts/test/TestPlan.md [hunk 6]
```diff
@@ -14,26 +14,28 @@
...
-    g. Try creating duplicate instrument for a candidate for a visit, should get an error message.
-    h. Try creating instrument for a candidate in a non-existing visit, should get an error message.
-    [Manual Testing]
+    The email can be pre-populated for each instrument using participant_emails table.  
+    c. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).  
+    d. Use the `Create Survey` button (no email address should be specified).  
+    e. Use the `Email survey` button by specifying email address.  
+    f. Check that the survey list shows the newly created survey with all the correct information.    
+    g. Try creating a survey for a candidate in a Project that the user is not affiliated with. This should not be allowed.  
+    h. Try mismatched PSCID and DCCID and should get an error message.
```
Comment (laemtl) [inline modules/survey_accounts/test/TestPlan.md:28]: 8. Try mismatched PSCID and DCCID and should get an error message.

File: modules/survey_accounts/test/TestPlan.md [hunk 7]
```diff
@@ -14,26 +14,28 @@
...
-    h. Try creating instrument for a candidate in a non-existing visit, should get an error message.
-    [Manual Testing]
+    The email can be pre-populated for each instrument using participant_emails table.  
+    c. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).  
+    d. Use the `Create Survey` button (no email address should be specified).  
+    e. Use the `Email survey` button by specifying email address.  
+    f. Check that the survey list shows the newly created survey with all the correct information.    
+    g. Try creating a survey for a candidate in a Project that the user is not affiliated with. This should not be allowed.  
+    h. Try mismatched PSCID and DCCID and should get an error message.  
+    i. Try creating duplicate instrument for a candidate for a visit, should get an error message.
```
Comment (laemtl) [inline modules/survey_accounts/test/TestPlan.md:29]: 9. Try creating duplicate instrument for a candidate for a visit, should get an error message.

File: modules/survey_accounts/test/TestPlan.md [hunk 8]
```diff
@@ -14,26 +14,28 @@
...
-    [Manual Testing]
+    The email can be pre-populated for each instrument using participant_emails table.  
+    c. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).  
+    d. Use the `Create Survey` button (no email address should be specified).  
+    e. Use the `Email survey` button by specifying email address.  
+    f. Check that the survey list shows the newly created survey with all the correct information.    
+    g. Try creating a survey for a candidate in a Project that the user is not affiliated with. This should not be allowed.  
+    h. Try mismatched PSCID and DCCID and should get an error message.  
+    i. Try creating duplicate instrument for a candidate for a visit, should get an error message.  
+    j. Try creating instrument for a candidate in a non-existing visit, should get an error message.
```
Comment (laemtl) [inline modules/survey_accounts/test/TestPlan.md:30]: 10. Try creating instrument for a candidate in a non-existing visit, should get an error message.

File: modules/survey_accounts/test/TestPlan.md [hunk 9]
```diff
@@ -14,26 +14,28 @@
...
+    [Manual Testing]  
 7.  Enable a new instrument for Survey module and verify survey can be created. (IsDirectEntry set to 1 in test_names table).
     [Manual Testing]
-8.  Verify Status column in the module updates appropriately
+8.  Verify Status column in the module updates appropriately  
     a. When survey is created Status should be set to `Created` when no email address is specified
```
Comment (laemtl) [inline modules/survey_accounts/test/TestPlan.md:35]: 1. When survey is created Status should be set to `Created` when no email address is specified

File: modules/survey_accounts/test/TestPlan.md [hunk 10]
```diff
@@ -14,26 +14,28 @@
...
-      and should be set to `Sent` if email address is specified
-    b. Status should be set to `In Progress` once the URL is accessed by user (or tester)
-    c. Status should be set to `Complete` once the survey is completed by clicking on Submit Data button on the review page.
+      and should be set to `Sent` if email address is specified  
+    b. Status should be set to `In Progress` once the URL is accessed by user (or tester)
```
Comment (laemtl) [inline modules/survey_accounts/test/TestPlan.md:37]: 2. Status should be set to `In Progress` once the URL is accessed by user (or tester)

File: modules/survey_accounts/test/TestPlan.md [hunk 11]
```diff
@@ -14,26 +14,28 @@
...
     [Manual Testing]
-8.  Verify Status column in the module updates appropriately
+8.  Verify Status column in the module updates appropriately  
     a. When survey is created Status should be set to `Created` when no email address is specified
-      and should be set to `Sent` if email address is specified
-    b. Status should be set to `In Progress` once the URL is accessed by user (or tester)
-    c. Status should be set to `Complete` once the survey is completed by clicking on Submit Data button on the review page.
+      and should be set to `Sent` if email address is specified  
+    b. Status should be set to `In Progress` once the URL is accessed by user (or tester)  
+    c. Status should be set to `Complete` once the survey is completed by clicking on Submit Data button on the review page.
```
Comment (laemtl) [inline modules/survey_accounts/test/TestPlan.md:38]: 3. Status should be set to `Complete` once the survey is completed by clicking on Submit Data button on the review page.

File: modules/survey_accounts/test/TestPlan.md [hunk 12]
```diff
@@ -14,26 +14,28 @@
...
-8.  Verify Status column in the module updates appropriately
+8.  Verify Status column in the module updates appropriately  
     a. When survey is created Status should be set to `Created` when no email address is specified
-      and should be set to `Sent` if email address is specified
-    b. Status should be set to `In Progress` once the URL is accessed by user (or tester)
-    c. Status should be set to `Complete` once the survey is completed by clicking on Submit Data button on the review page.
+      and should be set to `Sent` if email address is specified  
+    b. Status should be set to `In Progress` once the URL is accessed by user (or tester)  
+    c. Status should be set to `Complete` once the survey is completed by clicking on Submit Data button on the review page.  
 9.  Verify instrument is marked as `Complete` and Administration is set to `All` after step 8.c
```
Comment (laemtl) [inline modules/survey_accounts/test/TestPlan.md:39]: 9.  Verify instrument is marked as `Complete` and Administration is set to `All` after step 8.iii

**Comment by AlexandraLivadas** [comment]
@laemtl Including "sub-numbers" might be confusing (going from Step 6 to sub-step 1, etc). Can I make bullet points instead? This is done in some other test plans

File: modules/survey_accounts/test/TestPlan.md [hunk 13]
```diff
@@ -14,27 +14,29 @@
     Make sure that if you don’t fill out a field that is required,
     and you hit save and continue fields you have filled out are not erased.
     [Manual Testing]
-6.  Add Survey button:
-    a. Try sending the URL to yourself -  make sure that both email addresses match in order for the “Email Survey” button to work
-      (try mismatched email addresses to make sure that the "Email Survey" button remains inactive).
-    b. Once you hit “Email Survey” you should get a blank page where you can customize an email to go along with the URL –
+6.  Add Survey button:
```
Comment (driusan) [inline modules/survey_accounts/test/TestPlan.md:17]: can you fix the extra whitespace at the end of these lines that github is showing in the diff?

**Pull request merged**